src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1313,5 +1313,13 @@
   </data>
   <data name="CollectionsShouldImplementGenericInterfaceMultipleMessage" xml:space="preserve">
     <value>Type '{0}' directly or indirectly inherits '{1}' without implementing any of '{2}'. Publicly-visible types should implement the generic version to broaden usability.</value>
+  <data name="ConsiderPassingBaseTypesAsParametersDescription" xml:space="preserve">
```

Comment:
This was dropped when conflicts were fixed and is why the sarif file and markdown file produces empty descriptions currently.

```suggestion
  </data>
  <data name="ConsiderPassingBaseTypesAsParametersDescription" xml:space="preserve">
```

You'll need to re-run pack after making this change.

Feel free to make the changes in a single commit locally instead of accepting the suggestion from GitHub then running pack in a separate commit.

Reply:
Thank you @Youssef1313!